config-template.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copy this file to config.js and fill it in with your own information.
+ */
+module.exports = {
```

Comment:
Ah. Yeah I transferred the stuff from my secret.js file to this and it worked but I figured we needed a better way to document that process in the README. Maybe a specific section?